### ISSUE TITLE
task(name).clear does not clear descriptions associated with the actions being cleared

### ISSUE THREAD

**Issue by dchelimsky** [jimweirich/rake#88]
Given:

``` ruby
desc "the original foo"
task :foo do
end

task(:foo).clear

desc "a slightly different foo"
task :foo do
end
```

When/Then

```
$ rake -T foo
rake foo                          # the original foo / a slightly different foo
```

I expect "the original foo" description to go away when I say `task(:foo).clear`.

**Comment by rctay** [comment]
Fix in #97.

**Comment by jimweirich** [comment]
Merged #97, so this issue is obsolete.
